config/prod/aacr-interactome-s3website-new.yaml

```diff
@@ -0,0 +1,24 @@
+# Provision a S3 static website for AACR Interactome
+template_path: remote/managed-s3WebCloudfront.yaml
+stack_name: aacr-interactome-s3website-new
+parameters:
+  # The department for this resource (i.e. CompOnc, SysBio, Platform, etc..)
+  Department: "CompOnc"
+  # The project for this resource (i.e. NTAP, BSMN, DREAM, etc..)
+  Project: "AACR"
+  # The resource owner's email address
+  OwnerEmail: "xindi.guo@sagebase.org"
+  # Domain name for your website (sagebionetworks.org)
+  DomainName: "sagebionetworks.org"
+  # The sub domain name e.g. ('mysite' in mysite.sagebionetworks.org)
+  SubDomainName: "cancer"
+  # The Amazon Resource Name (ARN) of an AWS Certificate Manager (ACM) certificate
+  AcmCertificateArn: "arn:aws:acm:us-east-1:055273631518:certificate/1fdb2541-963b-4363-963b-69deb71637af"
```

Comment:
we have a different cert for sagebionetworks.org.  please replace with `arn:aws:acm:us-east-1:055273631518:certificate/62283392-710d-439e-ad40-3b53a6a78499`

Reply:
Thank you! It has been updated.